lib/beaker/shared/error_handler.rb

```diff
@@ -9,7 +9,7 @@ def report_and_raise(logger, e, msg)
       logger.pretty_backtrace(bt).each_line do |line|
         logger.error(line)
       end
-      raise e
+      raise e unless e.is_a? Beaker::DSL::FailTest
```

Comment:
You're right, I was playing with this, but thought I had removed it.. updated PR.